# Software

## Augustus

### Summary

AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.

**Version:** 3.2.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Bamtools

### Summary

C++ API & command-line toolkit for working with BAM data

**Version:** 2.4.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Bcftools

### Summary

Tools for dealing with BCF and VCF files

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Bedtools

### Summary

A powerful toolset for genome arithmetic

**Version:** 2.25.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Blast

### Summary

BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.

**Version:** 2.2.31

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Blat

### Summary

BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.

**Version:** 35

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Bowtie2

### Summary

Fast and sensitive read alignment

**Version:** 2.2.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Bwa

### Summary

The BWA read mapper.

**Version:** 0.7.15

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Circos

### Summary

Circos is a software package for visualizing data and information. It visualizes data in a circular layout

**Version:** 0.69.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Freebayes

### Summary

Bayesian haplotype-based polymorphism discovery and genotyping

**Version:** 1.0.2.29

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Genblasta

### Summary

genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.

**Version:** 1.0.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Genblastg

### Summary

genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.

**Version:** 1.38

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Gencore_annotation_docs

### Summary



**Version:** 1.0

**Conda Channel:** nyuad-cgsb

### HPC Modules

* gencore_annotation-1.0-0


## Hmmer

### Summary

Biosequence analysis using profile hidden Markov models

**Version:** 3.1b2

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Jellyfish

### Summary

Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence

**Version:** 2.2.3

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Kat

### Summary

KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts

**Version:** 2.0.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Kmergenie

### Summary

KmerGenie estimates the best k-mer length for genome de novo assembly

**Version:** 1.7016

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Pauda

### Summary



**Version:** 1.0.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Perl-app-cpanminus

### Summary

get, unpack, build and install modules from CPAN

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* test_1.0-None-0


## Perl-biox-workflow-plugin-filedetails

### Summary

Get metadata for files in directories processed by BioX::Workflow

**Version:** 0.11

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Perl-biox-workflow-plugin-fileexists

### Summary

a plugin to BioX::Workflow

**Version:** 0.13

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Perl-biox-workflow

### Summary

A very opinionated template based workflow writer.

**Version:** 1.10

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Perl-hpc-runner-command

### Summary



**Version:** 3.0.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Repeatmasker

### Summary

RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.

**Version:** 4.0.6

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Rmblast

### Summary

RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite.

**Version:** 2.2.28

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Samtools

### Summary

Tools for dealing with SAM, BAM and CRAM files

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Snpeff

### Summary

Genetic variant annotation and effect prediction toolbox

**Version:** 4.3

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Test_docs

### Summary



**Version:** 1.0

**Conda Channel:** default

### HPC Modules

* test_1.0-None-0


## Tophat

### Summary

A spliced read mapper for RNA-Seq

**Version:** 2.1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Transdecoder

### Summary

TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.

**Version:** 2.1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Trinotate

### Summary

Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms

**Version:** 3.0.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Vcflib

### Summary

Command-line tools for manipulating VCF files

**Version:** 1.0.0_rc1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


## Vcftools

### Summary

A set of tools written in Perl and C++ for working with VCF files.

**Version:** 0.1.14

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.0-0


