# Software

## Bcftools

### Summary

BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Bowtie2

### Summary

Fast and sensitive read alignment

**Version:** 2.2.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Bwa

### Summary

The BWA read mapper.

**Version:** 0.7.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Bzip2

### Summary



**Version:** 1.0.6

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-2.0


## Deeptools

### Summary

A set of user-friendly tools for normalization and visualzation of deep-sequencing data

**Version:** 2.3.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Fastqc

### Summary

A quality control tool for high throughput sequence data.

**Version:** 0.11.5

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Fastx_toolkit

### Summary



**Version:** 0.0.14

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Hisat2

### Summary

graph-based alignment of next generation sequencing reads to a population of genomes

**Version:** 2.0.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Kallisto

### Summary

Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.

**Version:** 0.44.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Multiqc

### Summary

Create aggregate bioinformatics analysis reports across many samples and tools

**Version:** 0.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Pyfaidx

### Summary

pyfaidx: efficient pythonic random access to fasta subsequences

**Version:** 0.4.7.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Pyfasta

### Summary

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files

**Version:** 0.5.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Pyfastaq

### Summary

Script to manipulate FASTA and FASTQ files, plus API for developers

**Version:** 3.11.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Python

### Summary

General purpose programming language.

**Version:** 3.5

**Conda Channel:** conda-forge

### HPC Modules

* gencore_qc-2.0


## Qualimap

### Summary

Quality control of alignment sequencing data and its derivatives like feature counts

**Version:** 2.2.2a

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


## Salmon

### Summary

Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments

**Version:** 0.10.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Samtools

### Summary

Tools for dealing with SAM, BAM and CRAM files

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Star

### Summary

An RNA-seq read aligner.

**Version:** 2.5.2a

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Tophat

### Summary

A spliced read mapper for RNA-Seq

**Version:** 2.1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-2.0


## Trimmomatic

### Summary

A flexible read trimming tool for Illumina NGS data

**Version:** 0.36

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-2.0


