# Software

## Abyss

### Summary

Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler

**Version:** 1.9.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Augustus

### Summary

AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.

**Version:** 3.2.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1


## Autoconf

### Summary

Extensible M4 macros that produce shell scripts to configure software source code packages.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1


## Automake

### Summary

Tool for automatically generating Makefile.in files compliant with the GNU Coding Standards.


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1


## Backcall

### Summary

Specifications for callback functions passed in to an API

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Backports.functools_lru_cache

### Summary

Backport of functools.lru_cache from Python 3.3 as published at ActiveState.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Backports

### Summary

A package to ensure the `backports` namespace is available.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Backports_abc

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Bamtools

### Summary

C++ API & command-line toolkit for working with BAM data

**Version:** 2.4.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Bcftools

### Summary

BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Bcftools

### Summary

BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Bedtools

### Summary

A powerful toolset for genome arithmetic

**Version:** 2.25.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Bioconductor-annotate

### Summary

Using R enviroments for annotation.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-annotationdbi

### Summary

Provides user interface and database connection code for annotation data packages using SQLite data storage.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-annotationhub

### Summary

This package provides a client for the Bioconductor AnnotationHub web resource. The AnnotationHub web resource provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-ballgown

### Summary

Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.

**Version:** 2.2.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Bioconductor-ballgown

### Summary

Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-biobase

### Summary

Functions that are needed by many other packages or which replace R functions.

**Version:** 2.32.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_variant_detection-1.1


## Bioconductor-biobase

### Summary

Functions that are needed by many other packages or which replace R functions.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_r-3.3.1
* gencore_rnaseq_analyze-1.0


## Bioconductor-biocgenerics

### Summary

S4 generic functions needed by many Bioconductor packages.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_base-1.1
* gencore_rnaseq_analyze-1.0


## Bioconductor-biocinstaller

### Summary

This package is used to install and update Bioconductor, CRAN, and (some) github packages.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-biocparallel

### Summary

This package provides modified versions and novel implementation of functions for parallel evaluation, tailored to use with Bioconductor objects.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-biomart

### Summary

In recent years a wealth of biological data has become available in public data repositories. Easy access to these valuable data resources and firm integration with data analysis is needed for comprehensive bioinformatics data analysis.  biomaRt provides an interface to a growing collection of databases implementing the BioMart software suite (http://www.biomart.org). The package enables retrieval of large amounts of data in a uniform way without the need to know the underlying database schemas or write complex SQL queries. Examples of BioMart databases are Ensembl, COSMIC, Uniprot, HGNC, Gramene, Wormbase and dbSNP mapped to Ensembl. These major databases give biomaRt users direct access to a diverse set of data and enable a wide range of powerful online queries from gene annotation to database mining.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-biostrings

### Summary

Memory efficient string containers, string matching algorithms, and other utilities, for fast manipulation of large biological sequences or sets of sequences.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-biovizbase

### Summary

The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.

**Version:** 1.18.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Bioconductor-biovizbase

### Summary

The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-bsgenome

### Summary

Infrastructure shared by all the Biostrings-based genome data packages.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-cummerbund

### Summary

Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.

**Version:** 2.12.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_malaria-1.1


## Bioconductor-cummerbund

### Summary

Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.

**Version:** 2.16.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Bioconductor-cummerbund

### Summary

Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-dada2

### Summary

The dada2 package infers exact amplicon sequence variants (ASVs) from high-throughput amplicon sequencing data, replacing the coarser and less accurate OTU clustering approach. The dada2 pipeline takes as input demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier, and genus-species assignment by exact matching.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_metagenomics-1.1


## Bioconductor-delayedarray

### Summary

Wrapping an array-like object (typically an on-disk object) in a DelayedArray object allows one to perform common array operations on it without loading the object in memory. In order to reduce memory usage and optimize performance, operations on the object are either delayed or executed using a block processing mechanism. Note that this also works on in-memory array-like objects like DataFrame objects (typically with Rle columns), Matrix objects, and ordinary arrays and data frames.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-deseq2

### Summary

Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.

**Version:** 1.10.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq_malaria-1.1


## Bioconductor-deseq2

### Summary

Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0


## Bioconductor-deseq2>

### Summary



**Version:** 1.12.4

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq-1.2


## Bioconductor-deseq

### Summary

Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution

**Version:** 1.22.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq_malaria-1.1


## Bioconductor-edger

### Summary

Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, Bisulfite-seq, SAGE and CAGE.

**Version:** 3.14.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1


## Bioconductor-edger

### Summary

Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, Bisulfite-seq, SAGE and CAGE.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genefilter

### Summary

Some basic functions for filtering genes

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-geneplotter

### Summary

Functions for plotting genomic data

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genomeinfodb

### Summary

Contains data and functions that define and allow translation between different chromosome sequence naming conventions (e.g., "chr1" versus "1"), including a function that attempts to place sequence names in their natural, rather than lexicographic, order.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genomeinfodbdata

### Summary

Files for mapping between NCBI taxonomy ID and species. Used by functions in the GenomeInfoDb package.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genomicalignments

### Summary

Provides efficient containers for storing and manipulating short genomic alignments (typically obtained by aligning short reads to a reference genome). This includes read counting, computing the coverage, junction detection, and working with the nucleotide content of the alignments.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genomicfeatures

### Summary

A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and cds of a given organism, from either the UCSC Genome Browser or a BioMart database (more sources will be supported in the future). This information is then stored in a local database that keeps track of the relationship between transcripts, exons, cds and genes. Flexible methods are provided for extracting the desired features in a convenient format.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-genomicranges

### Summary

The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages, respectively. Both packages build on top of the GenomicRanges infrastructure.

**Version:** 1.26.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2


## Bioconductor-genomicranges

### Summary

The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages, respectively. Both packages build on top of the GenomicRanges infrastructure.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0


## Bioconductor-gviz

### Summary

Genomic data analyses requires integrated visualization of known genomic information and new experimental data.  Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.

**Version:** 1.16.5

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Bioconductor-gviz

### Summary

Genomic data analyses requires integrated visualization of known genomic information and new experimental data.  Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-interactivedisplaybase

### Summary

The interactiveDisplayBase package contains the the basic methods needed to generate interactive Shiny based display methods for Bioconductor objects.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-iranges

### Summary

Provides efficient low-level and highly reusable S4 classes for storing, manipulating and aggregating over annotated ranges of integers. Implements an algebra of range operations, including efficient algorithms for finding overlaps and nearest neighbors. Defines efficient list-like classes for storing, transforming and aggregating large grouped data, i.e., collections of atomic vectors and DataFrames.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-limma

### Summary

Data analysis, linear models and differential expression for microarray data.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-rsamtools

### Summary

This package provides an interface to the 'samtools', 'bcftools', and 'tabix' utilities (see 'LICENCE') for manipulating SAM (Sequence Alignment / Map), FASTA, binary variant call (BCF) and compressed indexed tab-delimited (tabix) files.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-rtracklayer

### Summary

Extensible framework for interacting with multiple genome browsers (currently UCSC built-in) and manipulating annotation tracks in various formats (currently GFF, BED, bedGraph, BED15, WIG, BigWig and 2bit built-in). The user may export/import tracks to/from the supported browsers, as well as query and modify the browser state, such as the current viewport.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-s4vectors

### Summary

The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).

**Version:** 0.12.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2


## Bioconductor-s4vectors

### Summary

The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0


## Bioconductor-summarizedexperiment

### Summary

The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-sva

### Summary

The sva package contains functions for removing batch effects and other unwanted variation in high-throughput experiment. Specifically, the sva package contains functions for the identifying and building surrogate variables for high-dimensional data sets. Surrogate variables are covariates constructed directly from high-dimensional data (like gene expression/RNA sequencing/methylation/brain imaging data) that can be used in subsequent analyses to adjust for unknown, unmodeled, or latent sources of noise. The sva package can be used to remove artifacts in three ways: (1) identifying and estimating surrogate variables for unknown sources of variation in high-throughput experiments (Leek and Storey 2007 PLoS Genetics,2008 PNAS), (2) directly removing known batch effects using ComBat (Johnson et al. 2007 Biostatistics) and (3) removing batch effects with known control probes (Leek 2014 biorXiv). Removing batch effects and using surrogate variables in differential expression analysis have been shown to reduce dependence, stabilize error rate estimates, and improve reproducibility, see (Leek and Storey 2007 PLoS Genetics, 2008 PNAS or Leek et al. 2011 Nat. Reviews Genetics).

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-tximport

### Summary

Imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for different expression of gene-level counts.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-variantannotation

### Summary

Annotate variants, compute amino acid coding changes, predict coding outcomes.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-xvector

### Summary

Memory efficient S4 classes for storing sequences "externally" (behind an R external pointer, or on disk).

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Bioconductor-zlibbioc

### Summary

This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have these available via other means (most Linux and Mac users should have system-level access to zlib, and no direct need for this package). See the vignette for instructions on use.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Blast

### Summary

BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.

**Version:** 2.2.31

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_variant_detection-1.1


## Blat

### Summary

BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.

**Version:** 35

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Bleach

### Summary

An easy whitelist-based HTML-sanitizing tool.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Boost-cpp

### Summary

Free peer-reviewed portable C++ source libraries.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0


## Boost

### Summary

Free peer-reviewed portable C++ source libraries.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0


## Bowtie2

### Summary

Fast and sensitive read alignment

**Version:** 2.2.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Bowtie2

### Summary

Fast and sensitive read alignment

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Bwa

### Summary

The BWA read mapper.

**Version:** 0.7.15

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_variant_detection-1.1


## Bzip2

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Ca-certificates

### Summary

Certificates for use with other packages.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Cairo

### Summary

Cairo is a 2D graphics library with support for multiple output devices.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Cd-hit

### Summary

Clusters and compares protein or nucleotide sequences

**Version:** 4.6.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_metagenomics-1.1
* gencore_rad-1.1


## Certifi

### Summary

Python package for providing Mozilla's CA Bundle.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Circos

### Summary

Circos is a software package for visualizing data and information. It visualizes data in a circular layout

**Version:** 0.69.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1


## Cmake

### Summary

CMake is an extensible, open-source system that manages the build process

**Version:** 3.7.1

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1


## Cufflinks-patch

### Summary

Transcriptome assembly and differential expression analysis for RNA-Seq

**Version:** 2.2.1

**Conda Channel:** nyuad-cgsb

### HPC Modules

* gencore_rnaseq-1.2


## Cufflinks

### Summary

Transcriptome assembly and differential expression analysis for RNA-Seq.

**Version:** 2.2.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Cufflinks

### Summary

Transcriptome assembly and differential expression analysis for RNA-Seq.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3
* gencore_rnaseq_quant-1.0


## Curl

### Summary

tool and library for transferring data with URL syntax

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Cycler

### Summary

Composable style cycles

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Dbus

### Summary

Simple message bus system for applications to talk to one another

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Ddocent

### Summary

 dDocent is  an interactive bash wrapper to QC, assemble, map, and call SNPs from all types of RAD data

**Version:** 2.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rad_ddocent-1.1


## Decorator

### Summary

Better living through Python with decorators.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Deeptools

### Summary

A set of user-friendly tools for normalization and visualzation of deep-sequencing data

**Version:** 2.3.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Discovar

### Summary

Suitable for variant calling with reference and de novo assembly of small genomes.

**Version:** 52488

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_variant_detection-1.1


## Discovardenovo

### Summary

Suitable for de novo assembly of large and small genomes.

**Version:** 52488

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_variant_detection-1.1


## Entrypoints

### Summary

Discover and load entry points from installed packages

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Expat

### Summary

Expat XML parser library in C.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Fastqc

### Summary

A quality control tool for high throughput sequence data.

**Version:** 0.11.5

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Fastx_toolkit

### Summary



**Version:** 0.0.14

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Fontconfig

### Summary

A library for configuring and customizing font access

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Freebayes

### Summary

Bayesian haplotype-based polymorphism discovery and genotyping

**Version:** 1.0.2.29

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_rad-1.1
* gencore_variant_detection-1.1


## Freetype

### Summary

A Free, High-Quality, and Portable Font Engine

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Functools32

### Summary

Backport of the functools module from Python 3.2.3 for use on 2.7 and PyPy.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Futures

### Summary

Backport of the concurrent.futures package from Python 3.2

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Gatk

### Summary

The full Genome Analysis Toolkit (GATK) framework, license restricted.

**Version:** 3.5

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Gatk

### Summary

The full Genome Analysis Toolkit (GATK) framework, license restricted.

**Version:** 3.6

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Gcc

### Summary

The GNU Compiler Collection

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1


## Genblasta

### Summary

genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.

**Version:** 1.0.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Genblastg

### Summary

genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.

**Version:** 1.38

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Gettext

### Summary

Internationalization package.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Glib

### Summary

Provides core application building blocks for libraries and applications written in C.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Gmp

### Summary

The GNU multiprecision library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Graphite2

### Summary

A "smart font" system that handles the complexities of lesser-known languages of the world.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Gsl

### Summary

GNU Scientific Library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Gst-plugins-base

### Summary

GStreamer Base Plug-ins

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Gstreamer

### Summary

Library for constructing graphs of media-handling components

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Harfbuzz

### Summary

An OpenType text shaping engine.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Hdf5

### Summary

HDF5 is a data model, library, and file format for storing and managing data

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0


## Hisat2

### Summary

graph-based alignment of next generation sequencing reads to a population of genomes

**Version:** 2.0.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Hisat2

### Summary

graph-based alignment of next generation sequencing reads to a population of genomes

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Hmmer

### Summary

Biosequence analysis using profile hidden Markov models

**Version:** 3.1b2

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1


## Html5lib

### Summary

HTML parser based on the WHATWG HTML specification

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Htseq

### Summary

HTSeq is a Python library to facilitate processing and analysis of data from high-throughput sequencing (HTS) experiments.

**Version:** 0.6.1.post1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1


## Htseq

### Summary

HTSeq is a Python library to facilitate processing and analysis of data from high-throughput sequencing (HTS) experiments.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_quant-1.0


## Htslib

### Summary

C library for high-throughput sequencing data formats.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_quant-1.0


## Icu

### Summary

International Components for Unicode.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Idba

### Summary

IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads Sequencing data with Highly Uneven Sequencing Depth.

**Version:** 1.1.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Intel-openmp

### Summary



**Version:** latest

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Ipykernel

### Summary

IPython Kernel for Jupyter

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Ipython

### Summary

IPython: Productive Interactive Computing

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1
* gencore_rnaseq_analyze-1.0


## Ipython_genutils

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Jedi

### Summary

An autocompletion tool for Python that can be used for text editors.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Jellyfish

### Summary

Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence

**Version:** 2.2.3

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Jemalloc

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Jinja2

### Summary

An easy to use stand-alone template engine written in pure python.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Jpeg

### Summary

read/write jpeg COM, EXIF, IPTC medata

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Jsonschema

### Summary

An implementation of JSON Schema validation for Python

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Jupyter_client

### Summary

jupyter_client contains the reference implementation of the Jupyter protocol.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Jupyter_core

### Summary

Core common functionality of Jupyter projects.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Kallisto

### Summary

Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.

**Version:** 0.43.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq_malaria-1.1


## Kallisto

### Summary

Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3
* gencore_rnaseq_align-1.0


## Kiwisolver

### Summary

A fast implementation of the Cassowary constraint solver

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Kmergenie

### Summary

KmerGenie estimates the best k-mer length for genome de novo assembly

**Version:** 1.7016

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Kraken

### Summary

Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.

**Version:** 0.10.6_eaf8fb68

**Conda Channel:** bioconda

### HPC Modules

* gencore_kraken-1.1


## Krb5

### Summary

A network authentication protocol.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libdeflate

### Summary

libdeflate is a library for fast, whole-buffer DEFLATE-based compression and decompression.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_quant-1.0


## Libffi

### Summary

A Portable Foreign Function Interface Library

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libgcc-ng

### Summary

The GCC low-level runtime library

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libgcc

### Summary

Shared libraries for the GNU Compiler Collection

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libgfortran-ng

### Summary

The GNU Fortran Runtime Library

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Libgfortran

### Summary

Fortran libraries from the GNU Compiler Collection

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0


## Libiconv

### Summary

Provides iconv for systems which don't have one (or that cannot convert from/to Unicode.)

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libopenblas

### Summary



**Version:** latest

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Libpng

### Summary

PNG reference library

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libsodium

### Summary

A modern and easy-to-use crypto library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Libssh2

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libstdcxx-ng

### Summary

The GNU C++ Runtime Library

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libtiff

### Summary

Support for the Tag Image File Format (TIFF).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Libuuid

### Summary

Portable uuid C library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Libxcb

### Summary

This is the C-language Binding (XCB) package to the X Window System protocol

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Libxml2

### Summary

The XML C parser and toolkit of Gnome.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## M4

### Summary

Implementation of the traditional Unix macro processor.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1


## Macs2

### Summary

Model Based Analysis for ChIP-Seq data

**Version:** 2.1.1.20160309

**Conda Channel:** bioconda

### HPC Modules

* gencore_epigenetics-1.1


## Mafft

### Summary

Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform

**Version:** 7.221

**Conda Channel:** bioconda

### HPC Modules

* gencore_evonet-1.1


## Markupsafe

### Summary

A Python module that implements the jinja2.Markup string

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Matplotlib

### Summary

Publication quality figures in Python

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1
* gencore_rnaseq_quant-1.0


## Mawk

### Summary

mawk is an interpreter for the AWK Programming Language.

**Version:** 1.3.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1
* gencore_metagenomics-1.1


## Mcl

### Summary

MCL - a cluster algorithm for graphs

**Version:** 14.137

**Conda Channel:** bioconda

### HPC Modules

* gencore_evonet-1.1


## Metaphlan2

### Summary

MetaPhlAn is a computational tool for profiling the composition of microbial
communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic
shotgun sequencing data with species level resolution. From version 2.0
MetaPhlAn is also able to identify specific strains (in the not-so-frequent
cases in which the sample contains a previously sequenced strains) and to
track strains across samples for all species.


**Version:** 2.5.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1


## Metavelvet-sl-feature-extraction

### Summary

Perl libraries that do feature extraction for metavelvet-sl

**Version:** 1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1


## Metavelvet-sl

### Summary

MetaVelvet-SL : An extension of Velvet assembler to de novo metagenomic assembler utilizing supervised learning

**Version:** 1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1


## Metavelvet

### Summary

MetaVelvet : An extension of Velvet assembler to de novo metagenome assembly from short sequence reads

**Version:** 1.1.01

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1


## Mira

### Summary

MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)

**Version:** 4.0.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Mistune

### Summary

The fastest markdown parser in pure Python.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Mkl

### Summary



**Version:** latest

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Mkl_fft

### Summary

NumPy-based implementation of Fast Fourier Transform using Intel (R) Math Kernel Library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Mkl_random

### Summary

Intel (R) MKL-powered package for sampling from common probability distributions into NumPy arrays.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Mothur

### Summary

This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.

**Version:** 1.36.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_metagenomic-1.1
* gencore_metagenomics-1.1


## Multiqc

### Summary

Create aggregate bioinformatics analysis reports across many samples and tools

**Version:** 0.8

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Nbconvert

### Summary

Converting Jupyter Notebooks

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Nbformat

### Summary

The Jupyter Notebook format

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Ncurses

### Summary

Library for text-based user interfaces

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Notebook

### Summary

A web-based notebook environment for interactive computing

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Numpy-base

### Summary



**Version:** latest

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Numpy

### Summary

Array processing for numbers, strings, records, and objects.

**Version:** 1.11.1

**Conda Channel:** conda-forge

### HPC Modules

* gencore_epigenetics-1.1


## Numpy

### Summary

Array processing for numbers, strings, records, and objects.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Openssl

### Summary

OpenSSL is an open-source implementation of the SSL and TLS protocols

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Pandas

### Summary

High-performance, easy-to-use data structures and data analysis tools.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1


## Pandoc

### Summary

Universal markup converter (repackaged binaries)

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pandocfilters

### Summary

A python module for writing pandoc filters

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pango

### Summary

Text layout and rendering engine.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Parso

### Summary

A Python Parser

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pcre

### Summary

Regular expression pattern matching using the same syntax and semantics as Perl 5.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Pear

### Summary

paired-end read merger

**Version:** 0.9.6

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1
* gencore_variant_detection-1.1


## Perl-app-cpanminus

### Summary

get, unpack, build and install modules from CPAN

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_base-1.1
* gencore_biosails-1.1
* gencore_perl-1.1


## Perl-biox-workflow-command

### Summary

Opinionated Bioinformatics Genomics Workflow Creator

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_biosails-1.1


## Perl-dbi

### Summary

Database independent interface for Perl

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_perl-1.1


## Perl-hpc-runner-command

### Summary

Create composable bioinformatics hpc analyses.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_biosails-1.1


## Perl-termreadkey

### Summary

A perl module for simple terminal control

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_perl-1.1


## Perl-threaded

### Summary

The Perl language.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Perl

### Summary

The Perl programming language interpreter.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0


## Pexpect

### Summary

Pexpect makes Python a better tool for controlling other applications.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Picard

### Summary

Java tools for working with NGS data in the BAM format

**Version:** 2.0.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_malaria-1.1


## Picard

### Summary

Java tools for working with NGS data in the BAM format

**Version:** 2.5.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_variant_detection-1.1


## Pickleshare

### Summary

Tiny 'shelve'-like database with concurrency support

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pip

### Summary

PyPA recommended tool for installing Python packages

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_base-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Pixman

### Summary

A low-level software library for pixel manipulation

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Prinseq

### Summary

PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data

**Version:** 0.20.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Prompt_toolkit

### Summary

Library for building powerful interactive command lines in Python

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Ptyprocess

### Summary

Run a subprocess in a pseudo terminal

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pyfaidx

### Summary

pyfaidx: efficient pythonic random access to fasta subsequences

**Version:** 0.4.7.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Pyfasta

### Summary

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files

**Version:** 0.5.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Pyfastaq

### Summary

Script to manipulate FASTA and FASTQ files, plus API for developers

**Version:** 3.11.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Pygments

### Summary

Pygments is a generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications that need to prettify source code.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Pyparsing

### Summary

Create and execute simple grammars

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Pyqt

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Pysam

### Summary

Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_quant-1.0


## Python-dateutil

### Summary

Extensions to the standard Python datetime module.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Python

### Summary

General purpose programming language.

**Version:** 2.7

**Conda Channel:** conda-forge

### HPC Modules

* gencore_epigenetics-1.1


## Python

### Summary

General purpose programming language.

**Version:** 3.5

**Conda Channel:** conda-forge

### HPC Modules

* gencore_qc-1.1


## Python

### Summary

General purpose programming language.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Pytz

### Summary

World timezone definitions, modern and historical.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Pyzmq

### Summary

Python bindings for zeromq

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Qiime

### Summary

Quantitative Insights Into Microbial Ecology

**Version:** 1.9.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_qiime-1.1


## Qt

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Qualimap

### Summary

Quality control of alignment sequencing data and its derivatives like feature counts

**Version:** 2.2.2a

**Conda Channel:** bioconda

### HPC Modules

* gencore_qc-1.1


## Quast

### Summary

Quality Assessment Tool for Genome Assemblies

**Version:** 4.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_transcriptome-1.1


## R-acepack

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-assertthat

### Summary

assertthat is an extension to stopifnot() that makes it easy to declare the pre and
post conditions that you code should satisfy, while also producing friendly error
messages so that your users know what they've done wrong.


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-backports

### Summary

Implementations of functions which have been introduced in R since version 3.0.0.
The backports are conditionally exported which results in R resolving the function
names to the version shipped with R (if available) and uses the implemented backports
as fallback. This way package developers can make use of the new functions without
worrying about the minimum required R version.


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-base64enc

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-base

### Summary

R is a free software environment for statistical computing and graphics.

**Version:** 3.3.1

**Conda Channel:** r

### HPC Modules

* gencore_base-1.1
* gencore_variant_detection-1.1


## R-base

### Summary

R is a free software environment for statistical computing and graphics.

**Version:** latest

**Conda Channel:** r

### HPC Modules

* gencore_r-3.3.1
* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0


## R-bh

### Summary

Boost provides free peer-reviewed portable C++ source  libraries.  A large part of Boost is provided as C++ template code which is resolved entirely at compile-time without linking.  This  package aims to provide the most useful subset of Boost libraries  for template use among CRAN package. By placing these libraries in  this package, we offer a more efficient distribution system for CRAN  as replication of this code in the sources of other packages is  avoided. . As of release 1.60.0-2, the following Boost libraries are included: 'algorithm' 'any' 'bimap' 'bind' 'circular_buffer' 'concept' 'config' 'container' 'date'_'time' 'detail' 'dynamic_bitset' 'exception' 'filesystem' 'flyweight' 'foreach' 'functional' 'fusion' 'geometry' 'graph' 'heap' 'icl' 'integer' 'interprocess' 'intrusive' 'io' 'iostreams' 'iterator' 'math' 'move' 'mpl' 'multiprcecision' 'numeric' 'pending' 'phoenix' 'preprocessor' 'random' 'range' 'smart_ptr' 'spirit' 'tuple' 'type_trains' 'typeof' 'unordered' 'utility' 'uuid'.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-bindr

### Summary

Provides a simple interface for creating active bindings where the bound function accepts additional arguments.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-bindrcpp

### Summary

Provides an easy way to fill an environment with active bindings that call a C++ function.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-bitops

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-boot

### Summary

Functions and datasets for bootstrapping from the book "Bootstrap Methods and Their Application" by A. C. Davison and  D. V. Hinkley (1997, CUP), originally written by Angelo Canty for S.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-broom

### Summary

Summarizes key information about statistical objects in tidy tibbles. This makes it easy to report results, create plots and consistently work with large numbers of models at once. Broom provides three verbs that  each provide different types of information about a model. tidy() summarizes information about model components such as coefficients of a regression. glance() reports information about an entire model, such as goodness of fit measures like AIC and BIC. augment() adds information about individual observations to a dataset, such as fitted values or influence measures.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-cairo

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0


## R-car

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-caret

### Summary

Misc functions for training and plotting classification and regression models.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-catools

### Summary

Contains several basic utility functions including: moving (rolling, running) window statistic functions, read/write for GIF and ENVI binary files, fast calculation of AUC, LogitBoost classifier, base64 encoder/decoder, round-off-error-free sum and cumsum, etc.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-cellranger

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-checkmate

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-class

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-cli

### Summary

A suite of tools designed to build attractive command line interfaces ('CLIs'). Includes tools for drawing rules, boxes, trees, and 'Unicode' symbols with 'ASCII' alternatives.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-cluster

### Summary

Methods for Cluster analysis.  Much extended the original from Peter Rousseeuw, Anja Struyf and Mia Hubert, based on Kaufman and Rousseeuw (1990) "Finding Groups in Data".

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-codetools

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-colorspace

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-crayon

### Summary

Colored terminal output on terminals that support 'ANSI' color and highlight codes. It also works in 'Emacs' 'ESS'. 'ANSI' color support is automatically detected. Colors and highlighting can be combined and nested. New styles can also be created easily. This package was inspired by the 'chalk' 'JavaScript' project.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-curl

### Summary

The curl() and curl_download() functions provide highly configurable drop-in replacements for base url() and download.file() with better performance, support for encryption (https, ftps), gzip compression, authentication, and other 'libcurl' goodies. The core of the package implements a framework for performing fully customized requests where data can be processed either in memory, on disk, or streaming via the callback or connection interfaces. Some knowledge of 'libcurl' is recommended; for a more-user-friendly web client see the 'httr' package which builds on this package with http specific tools and logic.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-data.table

### Summary

Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, friendly and fast character-separated-value read/write. Offers a natural and flexible syntax, for faster development.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-dbi

### Summary

A database interface definition for communication between R and relational database management systems.  All classes in this package are virtual and need to be extended by the various R/DBMS implementations.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-dichromat

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-digest

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-dplyr

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-essentials

### Summary

Some essential packages for working with R

**Version:** latest

**Conda Channel:** r

### HPC Modules

* gencore_r-3.3.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_analyze-1.0
* gencore_variant_detection-1.1


## R-evaluate

### Summary

Parsing and evaluation tools that make it easy to recreate the command line behaviour of R.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-fastcluster

### Summary

This is a two-in-one package which provides interfaces to both R and Python. It implements fast hierarchical, agglomerative clustering routines. Part of the functionality is designed as drop-in replacement for existing routines: linkage() in the SciPy package 'scipy.cluster.hierarchy', hclust() in R's 'stats' package, and the 'flashClust' package. It provides the same functionality with the benefit of a much faster implementation. Moreover, there are memory-saving routines for clustering of vector data, which go beyond what the existing packages provide. For information on how to install the Python files, see the file INSTALL in the source distribution.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-forcats

### Summary

Helpers for reordering factor levels (including moving specified levels to front, ordering by first appearance, reversing, and randomly shuffling), and tools for modifying factor levels (including collapsing rare levels into other, 'anonymising', and manually 'recoding').

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-foreach

### Summary

Support for the foreach looping construct.  Foreach is an idiom that allows for iterating over elements in a collection, without the use of an explicit loop counter.  This package in particular is intended to be used for its return value, rather than for its side effects.  In that sense, it is similar to the standard lapply function, but doesn't require the evaluation of a function.  Using foreach without side effects also facilitates executing the loop in parallel.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-foreign

### Summary

Functions for reading and writing data stored by some versions of Epi Info, Minitab, S, SAS, SPSS, Stata, Systat and Weka and for reading and writing some dBase files.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-formatr

### Summary

Provides a function tidy_source() to format R source code. Spaces and indent will be added to the code automatically, and comments will be preserved under certain conditions, so that R code will be more human-readable and tidy. There is also a Shiny app as a user interface in this package (see tidy_app()).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-formula

### Summary

Infrastructure for extended formulas with multiple parts on the right-hand side and/or multiple responses on the left-hand side (see <DOI:10.18637/jss.v034.i01>).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-futile.logger

### Summary

Provides a simple yet powerful logging utility. Based loosely on log4j, futile.logger takes advantage of R idioms to make logging a convenient and easy to use replacement for cat and print statements.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-futile.options

### Summary

A scoped options management framework

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-gdata

### Summary

Various R programming tools for data manipulation, including

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-getopt

### Summary

Package designed to be used with Rscript to write ``#!'' shebang scripts that accept short and long flags/options. Many users will prefer using instead the packages optparse or argparse which add extra features like automatically generated help option and usage, support for default values, positional argument support, etc.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-ggplot2

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_base-1.1
* gencore_rnaseq_analyze-1.0


## R-gistr

### Summary

Work with ''GitHub'' ''gists'' from ''R'' (e.g.,  <http://en.wikipedia.org/wiki/GitHub#Gist>,  <https://help.github.com/articles/about-gists/>).
A ''gist'' is simply one or more files with code/text/images/etc. This package allows
the user to create new ''gists'', update ''gists'' with new files, rename files,
delete files, get and delete ''gists'', star and ''un-star'' ''gists'', fork ''gists'',
open a ''gist'' in your default browser, get embed code for a ''gist'', list ''gist''
''commits'', and get rate limit information when ''authenticated''. Some requests
require authentication and some do not.


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-glmnet

### Summary

Extremely efficient procedures for fitting the entire lasso or elastic-net regularization path for linear regression, logistic and multinomial regression models, Poisson regression and the Cox model. Two recent additions are the multiple-response Gaussian, and the grouped multinomial regression. The algorithm uses cyclical coordinate descent in a path-wise fashion, as described in the paper linked to via the URL below.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-glue

### Summary

An implementation of interpreted string literals, inspired by Python's Literal String Interpolation <https://www.python.org/dev/peps/pep-0498/> and Docstrings <https://www.python.org/dev/peps/pep-0257/> and Julia's Triple-Quoted String Literals <https://docs.julialang.org/en/stable/manual/strings/#triple-quoted-string-literals>.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-gplots

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-gridextra

### Summary

Provides a number of user-level functions to work with "grid" graphics, notably to arrange multiple grid-based plots on a page, and draw tables.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-gtable

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-gtools

### Summary

Functions to assist in R programming, including: - assist in developing, updating, and maintaining R and R packages ('ask', 'checkRVersion', 'getDependencies', 'keywords', 'scat'), - calculate the logit and inverse logit transformations ('logit', 'inv.logit'), - test if a value is missing, empty or contains only NA and NULL values ('invalid'), - manipulate R's .Last function ('addLast'), - define macros ('defmacro'), - detect odd and even integers ('odd', 'even'), - convert strings containing non-ASCII characters (like single quotes) to plain ASCII ('ASCIIfy'), - perform a binary search ('binsearch'), - sort strings containing both numeric and character components ('mixedsort'), - create a factor variable from the quantiles of a continuous variable ('quantcut'), - enumerate permutations and combinations ('combinations', 'permutation'), - calculate and convert between fold-change and log-ratio ('foldchange', 'logratio2foldchange', 'foldchange2logratio'), - calculate probabilities and generate random numbers from Dirichlet distributions ('rdirichlet', 'ddirichlet'), - apply a function over adjacent subsets of a vector ('running'), - modify the TCP\_NODELAY ('de-Nagle') flag for socket objects, - efficient 'rbind' of data frames, even if the column names don't match ('smartbind'), - generate significance stars from p-values ('stars.pval'), - convert characters to/from ASCII codes.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-haven

### Summary

Import foreign statistical formats into R via the embedded 'ReadStat' C library (https://github.com/WizardMac/ReadStat).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-hexbin

### Summary

Binning and plotting functions for hexagonal bins. Now uses and relies on grid graphics and formal (S4) classes and methods.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-highr

### Summary

Provides syntax highlighting for R source code. Currently it supports LaTeX and HTML
output. Source code of other languages is supported via Andre Simon's highlight
package (http://www.andre-simon.de).


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-hmisc

### Summary

Contains many functions useful for data analysis, high-level graphics, utility operations, functions for computing sample size and power, importing and annotating datasets, imputing missing values, advanced table making, variable clustering, character string manipulation, conversion of R objects to LaTeX and html code, and recoding variables.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-hms

### Summary

Implements an S3 class for storing and formatting time-of-day values, based on the 'difftime' class.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-htmltable

### Summary

Tables with state-of-the-art layout elements such as row spanners, column spanners, table spanners, zebra striping, and more. While allowing advanced layout, the underlying css-structure is simple in order to maximize compatibility with word processors such as 'MS Word' or 'LibreOffice'. The package also contains a few text formatting functions that help outputting text compatible with HTML/LaTeX.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-htmltools

### Summary

Tools for HTML generation and output.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-htmlwidgets

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-httpuv

### Summary

Provides low-level socket and protocol support for handling HTTP and WebSocket requests directly from within R.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-httr

### Summary

Useful tools for working with HTTP organised by HTTP verbs (GET(), POST(), etc). Configuration functions make it easy to control additional request components (authenticate(), add_headers() and so on).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-irdisplay

### Summary

An interface to the rich display capabilities of 'Jupyter' front-ends (e.g. 'Jupyter Notebook'). Designed to be used from a running 'IRkernel'.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-irkernel

### Summary

The R kernel for the 'Jupyter' environment executes R code which the front-end ('Jupyter Notebook' or other front-ends) submits to the kernel via the network.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-iterators

### Summary

Support for iterators, which allow a programmer to traverse through all the elements of a vector, list, or other collection of data.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-jsonlite

### Summary

A fast JSON parser and generator optimized for statistical data and the web. Started out as a fork of 'RJSONIO', but has been completely rewritten in recent versions. The package offers flexible, robust, high performance tools for working with JSON in R and is particularly powerful for building pipelines and interacting with a web API. The implementation is based on the mapping described in the vignette (Ooms, 2014). In addition to converting JSON data from/to R objects, 'jsonlite' contains functions to stream, validate, and prettify JSON data. The unit tests included with the package verify that all edge cases are encoded and decoded consistently for use with dynamic data in systems and applications.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-kernsmooth

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-knitr

### Summary

Provides a general-purpose tool for dynamic report generation in R using Literate Programming techniques.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-labeling

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-lambda.r

### Summary

A language extension to efficiently write functional programs in R. Syntax extensions include multi-part function definitions, pattern matching, guard statements, built-in (optional) type safety.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-lattice

### Summary

A powerful and elegant high-level data visualization system inspired by Trellis graphics, with an emphasis on multivariate data. Lattice is sufficient for typical graphics needs, and is also flexible enough to handle most nonstandard requirements. See ?Lattice for an introduction.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-latticeextra

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-lazyeval

### Summary

An alternative approach to non-standard evaluation using formulas. Provides a full implementation of LISP style 'quasiquotation', making it easier to generate code with other code.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-lme4

### Summary

Fit linear and generalized linear mixed-effects models. The models and their components
are represented using S4 classes and methods.  The core computational algorithms
are implemented using the 'Eigen' C++ library for numerical linear algebra and 'RcppEigen' "glue".


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-locfit

### Summary

Local regression, likelihood and density estimation.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-lubridate

### Summary

Functions to work with date-times and time-spans

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-magrittr

### Summary

Provides a mechanism for chaining commands with a new forward-pipe operator, %>%. This operator will forward a value, or the result of an expression, into the next function call/expression. There is flexible support for the type of right-hand side expressions. For more information, see package vignette. To quote Rene Magritte, "Ceci n'est pas un pipe."

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-maps

### Summary

Display of maps.  Projection code and larger maps are in separate packages ('mapproj' and 'mapdata').

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-markdown

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-mass

### Summary

Functions and datasets to support Venables and Ripley, "Modern Applied Statistics with S" (4th edition, 2002).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-matrix

### Summary

A rich hierarchy of matrix classes, including triangular, symmetric, and diagonal matrices, both dense and sparse and with pattern, logical and numeric entries.   Numerous methods for and operations on these matrices, using 'LAPACK' and 'SuiteSparse' libraries.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-matrixmodels

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-matrixstats

### Summary

High-performing functions operating on rows and columns of matrices, e.g. col / rowMedians(), col / rowRanks(), and col / rowSds().  Functions optimized per data type and for subsetted calculations such that both memory usage and processing time is minimized.  There are also optimized vector-based methods, e.g. binMeans(), madDiff() and weightedMedians().

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-memoise

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-mgcv

### Summary

GAMs, GAMMs and other generalized ridge regression with  multiple smoothing parameter estimation by GCV, REML or UBRE/AIC.  Includes a gam() function, a wide variety of smoothers, JAGS  support and distributions beyond the exponential family.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-mime

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-minqa

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-mnormt

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-modelmetrics

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-modelr

### Summary

Functions for modelling that help you seamlessly integrate modelling into a pipeline of data manipulation and visualisation.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-munsell

### Summary

Provides easy access to, and manipulation of, the Munsell  colours. Provides a mapping between Munsell's  original notation (e.g. "5R 5/10") and hexadecimal strings suitable  for use directly in R graphics. Also provides utilities  to explore slices through the Munsell colour tree, to transform  Munsell colours and display colour palettes.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-nlme

### Summary

Fit and compare Gaussian linear and nonlinear mixed-effects models.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-nloptr

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-nnet

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-openssl

### Summary

Bindings to OpenSSL libssl and libcrypto, plus custom SSH pubkey parsers. Supports RSA, DSA and EC curves P-256, P-384 and P-521. Cryptographic signatures can either be created and verified manually or via x509 certificates. AES can be used in cbc, ctr or gcm mode for symmetric encryption; RSA for asymmetric (public key) encryption or EC for Diffie Hellman. High-level envelope functions combine RSA and AES for encrypting arbitrary sized data. Other utilities include key generators, hash functions (md5, sha1, sha256, etc), base64 encoder, a secure random number generator, and 'bignum' math methods for manually performing crypto calculations on large multibyte integers.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-pbdzmq

### Summary

'ZeroMQ' is a well-known library for high-performance asynchronous messaging in scalable, distributed applications.  This package provides high level R wrapper functions to easily utilize 'ZeroMQ'. We mainly focus on interactive client/server programming frameworks. For convenience, a minimal 'ZeroMQ' library (4.2.2) is shipped with 'pbdZMQ', which can be used if no system installation of 'ZeroMQ' is available.  A few wrapper functions compatible with 'rzmq' are also provided.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-pbkrtest

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-pillar

### Summary

Provides a 'pillar' generic designed for formatting columns of data using the full range of colours provided by modern terminals.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-pkgconfig

### Summary

Set configuration options on a per-package basis. Options set by a given package only apply to that package, other packages are unaffected.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-plogr

### Summary

 A simple header-only logging library for C++. Add 'LinkingTo: plogr' to 'DESCRIPTION', and '#include <plogr.h>' in your C++ modules to use it.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-plyr

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-pryr

### Summary

Useful tools to pry back the covers of R and understand the language at a deeper level.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-psych

### Summary

A general purpose toolbox for personality, psychometric theory and experimental psychology.   Functions are primarily for multivariate analysis and scale construction using factor analysis, principal component analysis, cluster analysis and reliability analysis, although others provide basic descriptive statistics. Item Response Theory is done using  factor analysis of tetrachoric and polychoric correlations. Functions for analyzing data at multiple levels include within and between group statistics, including correlations and factor analysis.   Functions for simulating and testing particular item and test structures are included. Several functions  serve as a useful front end for structural equation modeling.  Graphical displays of path diagrams, factor analysis and structural equation models are created using basic graphics. Some of the functions are written to support a book on psychometric theory as well as publications in personality research. For more information, see the <http://personality-project.org/r> web page.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-purrr

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-quantmod

### Summary

Specify, build, trade, and analyse quantitative financial trading strategies.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-quantreg

### Summary

Estimation and inference methods for models of conditional quantiles:  Linear and nonlinear parametric and non-parametric (total variation penalized) models  for conditional quantiles of a univariate response and several methods for handling censored survival data.  Portfolio selection methods based on expected shortfall risk are also included.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-r6

### Summary

The R6 package allows the creation of classes with reference semantics, similar to R's built-in reference classes. Compared to reference classes, R6 classes are simpler and lighter-weight, and they are not built on S4 classes so they do not require the methods package. These classes allow public and private members, and they support inheritance, even when the classes are defined in different packages.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-randomforest

### Summary

Classification and regression based on a forest of trees using random inputs, based on Breiman (2001) <DOI:10.1023/A:1010933404324>.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rbokeh

### Summary

A native R plotting library that provides a flexible declarative interface for creating interactive web-based graphics, backed by the Bokeh visualization library <http://bokeh.pydata.org/>.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rcolorbrewer

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rcpp

### Summary

The 'Rcpp' package provides R functions as well as C++ classes which offer a seamless integration of R and C++. Many R data types and objects can be mapped back and forth to C++ equivalents which facilitates both writing of new code as well as easier integration of third-party libraries. Documentation about 'Rcpp' is provided by several vignettes included in this package, via the 'Rcpp Gallery' site at <http://gallery.rcpp.org>, the paper by Eddelbuettel and Francois (2011, <doi:10.18637/jss.v040.i08>), the book by Eddelbuettel (2013, <doi:10.1007/978-1-4614-6868-4>) and the paper by Eddelbuettel and Balamuta (2018, <doi:10.1080/00031305.2017.1375990>); see 'citation("Rcpp")' for details.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rcpparmadillo

### Summary

'Armadillo' is a templated C++ linear algebra library (by Conrad Sanderson) that aims towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions. Various matrix decompositions are provided through optional integration with LAPACK and ATLAS libraries. The 'RcppArmadillo' package includes the header files from the templated 'Armadillo' library. Thus users do not need to install 'Armadillo' itself in order to use 'RcppArmadillo'. From release 7.800.0 on, 'Armadillo' is licensed under Apache License 2; previous releases were under licensed as MPL 2.0 from version 3.800.0 onwards and LGPL-3 prior to that; 'RcppArmadillo' (the 'Rcpp' bindings/bridge to Armadillo) is licensed under the GNU GPL version 2 or later, as is the rest of 'Rcpp'. Note that Armadillo requires a fairly recent compiler; for the g++ family at least version 4.6.* is required. 

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rcppeigen

### Summary

R and ''Eigen'' integration using ''Rcpp''. ''Eigen'' is a C++ template library for
linear algebra: matrices, vectors, numerical solvers and related algorithms.  It
supports dense and sparse matrices on integer, floating point and complex numbers,
decompositions of such matrices, and solutions of linear systems. Its performance
on many algorithms is comparable with some of the best implementations based on
''Lapack'' and level-3 ''BLAS''. . The ''RcppEigen'' package includes the header
files from the ''Eigen'' C++ template library (currently version 3.2.9). Thus users
do not need to install ''Eigen'' itself in order to use ''RcppEigen''. . Since version
3.1.1, ''Eigen'' is licensed under the Mozilla Public License (version 2); earlier
version were licensed under the GNU LGPL version 3 or later. ''RcppEigen'' (the
''Rcpp'' bindings/bridge to ''Eigen'') is licensed under the GNU GPL version 2 or
later, as is the rest of ''Rcpp''


**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rcurl

### Summary

A wrapper for libcurl. Provides functions to allow one to compose general HTTP requests and provides convenient functions to fetch URIs, get & post forms, etc. and process the results returned by the Web server. This provides a great deal of control over the HTTP/FTP/... connection and the form of the request while providing a higher-level interface than is available just using R socket connections.  Additionally, the underlying implementation is robust and extensive, supporting FTP/FTPS/TFTP (uploads and downloads), SSL/HTTPS, telnet, dict, ldap, and also supports cookies, redirects, authentication, etc.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-readr

### Summary

Read flat/tabular text files from disk (or a connection).

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-readxl

### Summary

Import excel files into R. Supports '.xls' via the embedded 'libxls' C library <https://sourceforge.net/projects/libxls/> and '.xlsx' via the embedded 'RapidXML' C++ library <https://rapidxml.sourceforge.net>. Works on Windows, Mac and Linux without external dependencies.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-recommended

### Summary

R is a free software environment for statistical computing and graphics.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rematch

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-repr

### Summary

String and binary representations of objects for several formats / mime types.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-reshape2

### Summary

Flexibly restructure and aggregate data using just two functions: melt and 'dcast' (or 'acast').

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rjson

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rlang

### Summary

A toolbox for working with base types, core R features like the condition system, and core 'Tidyverse' features like tidy evaluation.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rmarkdown

### Summary

Convert R Markdown documents into a variety of formats.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rpart

### Summary

Recursive partitioning for classification,  regression and survival trees.  An implementation of most of the  functionality of the 1984 book by Breiman, Friedman, Olshen and Stone.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rprojroot

### Summary

Robust, reliable and flexible paths to files below a project root. The 'root' of a project is defined as a directory that matches a certain criterion, e.g., it contains a certain regular file.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rsqlite

### Summary

Embeds the 'SQLite' database engine in R and provides an interface compliant with the 'DBI' package. The source for the 'SQLite' engine is included.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-rvest

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-scales

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-selectr

### Summary

Translates a CSS3 selector into an equivalent XPath expression. This allows us to use CSS selectors when working with the XML package as it can only evaluate XPath expressions. Also provided are convenience functions useful for using CSS selectors on XML nodes. This package is a port of the Python package 'cssselect' (<https://cssselect.readthedocs.io/>).

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-shiny

### Summary

Makes it incredibly easy to build interactive web applications with R. Automatic "reactive" binding between inputs and outputs and extensive prebuilt widgets make it possible to build beautiful, responsive, and powerful applications with minimal effort.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-snow

### Summary

Support for simple parallel computing in R.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-sourcetools

### Summary

Tools for the reading and tokenization of R code. The 'sourcetools' package provides both an R and C++ interface for the tokenization of R code, and helpers for interacting with the tokenized representation of R code.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-sparsem

### Summary

Some basic linear algebra functionality for sparse matrices is provided:  including Cholesky decomposition and backsolving as well as  standard R subsetting and Kronecker products.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-spatial

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-stringi

### Summary

Allows for fast, correct, consistent, portable, as well as convenient character string/text processing in every locale and any native encoding. Owing to the use of the 'ICU' library, the package provides 'R' users with platform-independent functions known to 'Java', 'Perl', 'Python', 'PHP', and 'Ruby' programmers. Available features include: pattern searching (e.g., with 'Java'-like regular expressions or the 'Unicode' collation algorithm), random string generation, case mapping, string transliteration, concatenation, Unicode normalization, date-time formatting and parsing, and many more.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-stringr

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-survival

### Summary

Contains the core survival analysis routines, including definition of Surv objects,  Kaplan-Meier and Aalen-Johansen (multi-state) curves, Cox models, and parametric accelerated failure time models.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-tibble

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-tidyr

### Summary

An evolution of 'reshape2'. It's designed specifically for data tidying (not general reshaping or aggregating) and works well with 'dplyr' data pipelines.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-tidyselect

### Summary

A backend for the selecting functions of the 'tidyverse'. It makes it easy to implement select-like functions in your own packages in a way that is consistent with other 'tidyverse' interfaces for selection.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-tidyverse

### Summary

The 'tidyverse' is a set of packages that work in harmony because they share common data representations and 'API' design. This package is designed to make it easy to install and load multiple 'tidyverse' packages in a single step. Learn more about the 'tidyverse' at <https://github.com/hadley/tidyverse>.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-ttr

### Summary

Functions and data to construct technical trading rules with R.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-utf8

### Summary

Process and print 'UTF-8' encoded international text (Unicode). Input, validate, normalize, encode, format, and display.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-uuid

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-viridis

### Summary

Port of the new 'matplotlib' color maps ('viridis' - the default -, 'magma', 'plasma' and 'inferno') to 'R'. 'matplotlib' <http://matplotlib.org/> is a popular plotting library for 'python'. These color maps are designed in such a way that they will analytically be perfectly perceptually-uniform, both in regular form and also when converted to black-and-white. They are also designed to be perceived by readers with the most common form of color blindness.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-viridislite

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-xml2

### Summary

Work with XML files using a simple, consistent interface. Built on top of the 'libxml2' C library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-xml

### Summary

Many approaches for both reading and creating XML (and HTML) documents (including DTDs), both local and accessible via HTTP or FTP.  Also offers access to an 'XPath' "interpreter".

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-xtable

### Summary

Coerce data to LaTeX and HTML tables.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-xts

### Summary

Provide for uniform handling of R's different time-based data classes by extending zoo, maximizing native format information preservation and allowing for user level customization and extension, while simplifying cross-class interoperability.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-yaml

### Summary

This package implements the libyaml YAML 1.1 parser and emitter (http://pyyaml.org/wiki/LibYAML) for R.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R-zoo

### Summary

An S3 class with methods for totally ordered indexed observations. It is particularly aimed at irregular time series of numeric vectors/matrices and factors. zoo's key design goals are independence of a particular index/date/time class and consistency with ts and base R by providing methods to extend standard generics.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## R

### Summary

R is a free software environment for statistical computing and graphics.

**Version:** latest

**Conda Channel:** r

### HPC Modules

* gencore_r-3.3.1
* gencore_rnaseq-1.2
* gencore_rnaseq_analyze-1.0


## Rainbow

### Summary

Efficient tool for clustering and assembling short reads, especially for RAD

**Version:** 2.0.4

**Conda Channel:** bioconda

### HPC Modules

* gencore_rad-1.1


## Readline

### Summary

library for editing command lines as they are typed in

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Repeatmasker

### Summary

RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.

**Version:** 4.0.6

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1


## Rmblast

### Summary

RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite.

**Version:** 2.2.28

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1


## Rsem

### Summary

RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data. The RSEM package provides an user-friendly interface, supports threads for parallel computation of the EM algorithm, single-end and paired-end read data, quality scores, variable-length reads and RSPD estimation. In addition, it provides posterior mean and 95% credibility interval estimates for expression levels. For visualization, It can generate BAM and Wiggle files in both transcript-coordinate and genomic-coordinate. Genomic-coordinate files can be visualized by both UCSC Genome browser and Broad Institute's Integrative Genomics Viewer (IGV). Transcript-coordinate files can be visualized by IGV. RSEM also has its own scripts to generate transcript read depth plots in pdf format. The unique feature of RSEM is, the read depth plots can be stacked, with read depth contributed to unique reads shown in black and contributed to multi-reads shown in red. In addition, models learned from data can also be visualized. Last but not least, RSEM contains a simulator.

**Version:** 1.2.28

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_trinity-1.1


## Salmon

### Summary

Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments

**Version:** 0.6.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1


## Salmon

### Summary

Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Samtools

### Summary

Tools for dealing with SAM, BAM and CRAM files

**Version:** 1.3.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Samtools

### Summary

Tools for dealing with SAM, BAM and CRAM files

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_quant-1.0


## Scipy

### Summary

Scientific Library for Python

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_python-1.1


## Send2trash

### Summary

Python library to natively send files to Trash (or Recycle bin) on all platforms.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Seqtk

### Summary

Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format

**Version:** 1.2

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Setuptools

### Summary

Download, build, install, upgrade, and uninstall Python packages

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_base-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Sickle

### Summary

A lightweight OAI client library for Python

**Version:** 0.5

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_transcriptome-1.1


## Simplegeneric

### Summary

Simple generic functions (similar to Python's own len(), pickle.dump(), etc.)

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Singledispatch

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Sip

### Summary

Create Python bindings for C and C++ libraries

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Six

### Summary

Python 2 and 3 compatibility utilities

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Snpeff

### Summary

Genetic variant annotation and effect prediction toolbox

**Version:** 4.3

**Conda Channel:** bioconda

### HPC Modules

* gencore_variant_detection-1.1


## Snpsift

### Summary

Toolbox that allows you to filter and manipulate annotated files

**Version:** 4.3

**Conda Channel:** bioconda

### HPC Modules

* gencore_variant_detection-1.1


## Soapdenovo2-gapcloser

### Summary

a tool named GapCloser for SOAPdenovo.

**Version:** 1.12

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic_soapdenovo2-1.1


## Soapdenovo2-prepare

### Summary

SoapDenovo2 Data prepare module using assembled contig to do scaffold assembly

**Version:** 2.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic_soapdenovo2-1.1


## Soapdenovo2

### Summary

SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for the human-sized genomes.

**Version:** 2.40

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic_soapdenovo2-1.1


## Spades

### Summary

SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.


**Version:** 3.9.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Sqlite

### Summary

Implements a self-contained, zero-configuration, SQL database engine

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Stacks

### Summary

Stacks is a software pipeline for building loci from RAD-seq

**Version:** 1.40

**Conda Channel:** bioconda

### HPC Modules

* gencore_rad-1.1


## Star

### Summary

An RNA-seq read aligner.

**Version:** 2.5.2a

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Star

### Summary

An RNA-seq read aligner.

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Stringtie

### Summary

Transcriptome assembly and quantification for RNA-seq

**Version:** 1.3.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Stringtie

### Summary

Transcriptome assembly and quantification for RNA-seq

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_quant-1.0


## Subprocess32

### Summary

A backport of the subprocess module from Python 3.2/3.3 for use on 2.x.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_quant-1.0


## Tbb

### Summary



**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Terminado

### Summary

Terminals served by tornado websockets

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Testpath

### Summary

Testpath is a collection of utilities for Python code working with files and commands.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Tk

### Summary

A dynamic programming language with GUI support.  Bundles Tcl and Tk.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Tophat

### Summary

A spliced read mapper for RNA-Seq

**Version:** 2.1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Tophat

### Summary

A spliced read mapper for RNA-Seq

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_align-1.0


## Tornado

### Summary

A Python web framework and asynchronous networking library, originally developed at FriendFeed.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Traitlets

### Summary

configuration system for Python applications

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Transdecoder

### Summary

TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.

**Version:** 2.1.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_trinity-1.1


## Trimmomatic

### Summary

A flexible read trimming tool for Illumina NGS data

**Version:** 0.35

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq_malaria-1.1


## Trimmomatic

### Summary

A flexible read trimming tool for Illumina NGS data

**Version:** 0.36

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_qc-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3


## Trinity

### Summary

Trinity RNA-Seq de novo transcriptome assembly

**Version:** 2.2.0

**Conda Channel:** bioconda

### HPC Modules

* gencore_trinity-1.1


## Trinotate

### Summary

Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms

**Version:** 3.0.1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_trinity-1.1


## Vcflib

### Summary

Command-line tools for manipulating VCF files

**Version:** 1.0.0_rc1

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Vcflib

### Summary

Command-line tools for manipulating VCF files

**Version:** latest

**Conda Channel:** bioconda

### HPC Modules

* gencore_rnaseq-1.3


## Vcftools

### Summary

A set of tools written in Perl and C++ for working with VCF files. This package only contains the C++ libraries whereas the package perl-vcftools-vcf contains the perl libraries

**Version:** 0.1.14

**Conda Channel:** bioconda

### HPC Modules

* gencore_annotation-1.1
* gencore_de_novo_genomic-1.1
* gencore_de_novo_metagenomic-1.1
* gencore_de_novo_transcriptome-1.1
* gencore_epigenetics-1.1
* gencore_metagenomics-1.1
* gencore_rad-1.1
* gencore_rnaseq-1.2
* gencore_rnaseq-1.3
* gencore_rnaseq_malaria-1.1
* gencore_variant_detection-1.1


## Velvet

### Summary

Sequence Assembler for short reads

**Version:** 1.2.10

**Conda Channel:** bioconda

### HPC Modules

* gencore_de_novo_genomic-1.1


## Wcwidth

### Summary

Measures number of Terminal column cells of wide-character codes

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Webencodings

### Summary

Character encoding aliases for legacy web content

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Wget

### Summary



**Version:** latest

**Conda Channel:** default

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Wheel

### Summary

A built-package format for Python.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Xorg-kbproto

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-libice

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-libsm

### Summary

The X.org core session management library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-libx11

### Summary

The main client library for X Windows version 11.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-libxau

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Xorg-libxdmcp

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Xorg-libxext

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-libxrender

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-renderproto

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-xextproto

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xorg-xproto

### Summary



**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Xz

### Summary

Data compression software with high compression ratio

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


## Zeromq

### Summary

A high-performance asynchronous messaging library.

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_rnaseq_analyze-1.0


## Zlib

### Summary

Massively spiffy yet delicately unobtrusive compression library

**Version:** latest

**Conda Channel:** conda-forge

### HPC Modules

* gencore_build-1.1
* gencore_rnaseq_align-1.0
* gencore_rnaseq_analyze-1.0
* gencore_rnaseq_quant-1.0


