# Software

| | gencore_aligners-2.0 | gencore_qc-2.0 | gencore_rnaseq_align-2.0 | gencore_variant_detection-2.0 |
| --- | --- | --- | --- | ---  |
| bamtools=2.4.0 | | | | **Y** |
| bcftools=1.3.1 | **Y** | | **Y** | **Y** |
| bedtools=2.25.0 | | | | **Y** |
| bioconductor-biobase=2.32.0 | | | | **Y** |
| blast=2.2.31 | | | | **Y** |
| blat=35 | | | | **Y** |
| bowtie2=2.2.8 | **Y** | | **Y** | |
| bwa=0.7.4 | **Y** | | **Y** | |
| bzip2=1.0.6 | | | **Y** | |
| deeptools=2.3.2 | | **Y** | | |
| discovar=52488 | | | | **Y** |
| discovardenovo=52488 | | | | **Y** |
| ensembl-vep=latest | | | | **Y** |
| fastqc=0.11.5 | | **Y** | | |
| fastx_toolkit=0.0.14 | | **Y** | | |
| freebayes=1.0.2.29 | | | | **Y** |
| gatk=3.5 | | | | **Y** |
| hisat2=2.0.4 | **Y** | | **Y** | |
| kallisto=0.44.0 | **Y** | | **Y** | |
| multiqc=0.8 | | **Y** | | |
| pear=0.9.6 | | | | **Y** |
| picard=2.5.0 | | | | **Y** |
| prinseq=0.20.4 | | | | **Y** |
| pyfaidx=0.4.7.1 | | **Y** | | |
| pyfasta=0.5.2 | | **Y** | | |
| pyfastaq=3.11.0 | | **Y** | | |
| python=3.5 | | **Y** | | |
| qualimap=2.2.2a | | **Y** | | |
| r-base=3.3.1 | | | | **Y** |
| r-essentials=latest | | | | **Y** |
| salmon=0.10.2 | **Y** | | **Y** | |
| samtools=1.3.1 | **Y** | | **Y** | **Y** |
| seqtk=1.2 | | | | **Y** |
| snpeff=4.3 | | | | **Y** |
| snpsift=4.3 | | | | **Y** |
| star=2.5.2a | **Y** | | **Y** | |
| tophat=2.1.0 | **Y** | | **Y** | |
| trimmomatic=0.36 | | **Y** | | |
| vcflib=1.0.0_rc1 | | | | **Y** |
| vcftools=0.1.14 | | | | **Y** |

