# gencore_qc_1.0
## Summary

Coming soon!

## Software Packages

### deeptools
**Version:** 2.3.2

**Conda Channel:** bioconda

#### Summary:
A set of user-friendly tools for normalization and visualzation of deep-sequencing data



### fastqc
**Version:** 0.11.5

**Conda Channel:** bioconda

#### Summary:
A quality control tool for high throughput sequence data.



### fastx_toolkit
**Version:** 0.0.14

**Conda Channel:** bioconda

#### Summary:




### gencore_qc_docs
**Version:** 1.0

**Conda Channel:** nyuad-cgsb

#### Summary:




### multiqc
**Version:** 0.8

**Conda Channel:** bioconda

#### Summary:
Create aggregate bioinformatics analysis report across many samples



### perl-biox-workflow-plugin-filedetails
**Version:** 0.11

**Conda Channel:** bioconda

#### Summary:
Get metadata for files in directories processed by BioX::Workflow



### perl-biox-workflow-plugin-fileexists
**Version:** 0.13

**Conda Channel:** bioconda

#### Summary:
a plugin to BioX::Workflow



### perl-biox-workflow
**Version:** 1.10

**Conda Channel:** bioconda

#### Summary:
A very opinionated template based workflow writer.



### perl-hpc-runner-slurm
**Version:** 2.58

**Conda Channel:** bioconda

#### Summary:
Job Submission to Slurm



### pyfaidx
**Version:** 0.4.7.1

**Conda Channel:** bioconda

#### Summary:
pyfaidx: efficient pythonic random access to fasta subsequences



### pyfasta
**Version:** 0.5.2

**Conda Channel:** bioconda

#### Summary:
fast, memory-efficient, pythonic (and command-line) access to fasta sequence files



### pyfastaq
**Version:** 3.11.0

**Conda Channel:** bioconda

#### Summary:
Script to manipulate FASTA and FASTQ files, plus API for developers



### python
**Version:** 3.5

**Conda Channel:** conda-forge

#### Summary:
General purpose programming language



### trimmomatic
**Version:** 0.36

**Conda Channel:** bioconda

#### Summary:
Genetic variant annotation and effect prediction toolbox



