# gencore_de_novo_transcriptome
## Summary

Coming soon!

## Software Packages

### augustus
**Version:** 3.2.2

**Conda Channel:** bioconda

#### Summary:
AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.



### bamtools
**Version:** 2.4.0

**Conda Channel:** bioconda

#### Summary:
C++ API & command-line toolkit for working with BAM data



### bcftools
**Version:** 1.3.1

**Conda Channel:** bioconda

#### Summary:
BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.



### bedtools
**Version:** 2.25.0

**Conda Channel:** bioconda

#### Summary:
A powerful toolset for genome arithmetic



### bioconductor-deseq2
**Version:** 1.10.1

**Conda Channel:** bioconda

#### Summary:
Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.



### bioconductor-deseq
**Version:** 1.22.1

**Conda Channel:** bioconda

#### Summary:
Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution



### bioconductor-edger
**Version:** 3.14.0

**Conda Channel:** bioconda

#### Summary:
Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, Bisulfite-seq, SAGE and CAGE.



### blast
**Version:** 2.2.31

**Conda Channel:** bioconda

#### Summary:
BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.



### blat
**Version:** 35

**Conda Channel:** bioconda

#### Summary:
BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.



### bowtie2
**Version:** 2.2.8

**Conda Channel:** bioconda

#### Summary:
Fast and sensitive read alignment



### bwa
**Version:** 0.7.15

**Conda Channel:** bioconda

#### Summary:
The BWA read mapper.



### circos
**Version:** 0.69.2

**Conda Channel:** bioconda

#### Summary:
Circos is a software package for visualizing data and information. It visualizes data in a circular layout



### cufflinks
**Version:** 2.2.1

**Conda Channel:** bioconda

#### Summary:
Transcriptome assembly and differential expression analysis for RNA-Seq.



### hmmer
**Version:** 3.1b2

**Conda Channel:** bioconda

#### Summary:
Biosequence analysis using profile hidden Markov models



### htseq
**Version:** 0.6.1.post1

**Conda Channel:** bioconda

#### Summary:
HTSeq is a Python library to facilitate processing and analysis of data from high-throughput sequencing (HTS) experiments.



### pear
**Version:** 0.9.6

**Conda Channel:** bioconda

#### Summary:
paired-end read merger



### picard
**Version:** 2.5.0

**Conda Channel:** bioconda

#### Summary:
Java tools for working with NGS data in the BAM format



### prinseq
**Version:** 0.20.4

**Conda Channel:** bioconda

#### Summary:
PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data



### quast
**Version:** 4.1

**Conda Channel:** bioconda

#### Summary:
Quality Assessment Tool for Genome Assemblies



### rsem
**Version:** 1.2.28

**Conda Channel:** bioconda

#### Summary:
RSEM (RNA-Seq by Expectation-Maximization)



### salmon
**Version:** 0.6.0

**Conda Channel:** bioconda

#### Summary:
Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments



### samtools
**Version:** 1.3.1

**Conda Channel:** bioconda

#### Summary:
Tools for dealing with SAM, BAM and CRAM files



### seqtk
**Version:** 1.2

**Conda Channel:** bioconda

#### Summary:
Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format



### sickle
**Version:** 0.5

**Conda Channel:** bioconda

#### Summary:
A lightweight OAI client library for Python



### star
**Version:** 2.5.2a

**Conda Channel:** bioconda

#### Summary:
An RNA-seq read aligner.



### tophat
**Version:** 2.1.0

**Conda Channel:** bioconda

#### Summary:
A spliced read mapper for RNA-Seq



### trimmomatic
**Version:** 0.36

**Conda Channel:** bioconda

#### Summary:
Genetic variant annotation and effect prediction toolbox



### vcflib
**Version:** 1.0.0_rc1

**Conda Channel:** bioconda

#### Summary:
Command-line tools for manipulating VCF files



### vcftools
**Version:** 0.1.14

**Conda Channel:** bioconda

#### Summary:
A set of tools written in Perl and C++ for working with VCF files. This package only contains the C++ libraries whereas the package perl-vcftools-vcf contains the perl libraries



