# gencore_rnaseq
## Summary

Coming soon!

## Software Packages

### bamtools
**Version:** 2.4.0

**Conda Channel:** bioconda

#### Summary:
C++ API & command-line toolkit for working with BAM data



### bcftools
**Version:** 1.3.1

**Conda Channel:** bioconda

#### Summary:
Tools for dealing with BCF and VCF files



### bedtools
**Version:** 2.25.0

**Conda Channel:** bioconda

#### Summary:
A powerful toolset for genome arithmetic



### bioconductor-ballgown
**Version:** 2.2.0

**Conda Channel:** bioconda

#### Summary:




### bioconductor-biovizbase
**Version:** 1.18.0

**Conda Channel:** bioconda

#### Summary:
The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.



### bioconductor-cummerbund
**Version:** 2.16.0

**Conda Channel:** bioconda

#### Summary:
Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.



### bioconductor-deseq2
**Version:** 1.12.4

**Conda Channel:** bioconda

#### Summary:
Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.



### bioconductor-edger
**Version:** 3.14.0

**Conda Channel:** bioconda

#### Summary:
Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.



### bioconductor-genomicranges
**Version:** 1.26.1

**Conda Channel:** bioconda

#### Summary:
The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyze high-throughput sequencing data (a.k.a. NGS data). The package defines general purpose containers for storing genomic intervals. Specialized containers for representing and manipulating short alignments against a reference genome are defined in the GenomicAlignments package.



### bioconductor-gviz
**Version:** 1.16.5

**Conda Channel:** bioconda

#### Summary:
Genomic data analyses requires integrated visualization of known genomic information and new experimental data. Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.



### bioconductor-s4vectors
**Version:** 0.12.0

**Conda Channel:** bioconda

#### Summary:
The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).



### blat
**Version:** 35

**Conda Channel:** bioconda

#### Summary:
BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.



### bowtie2
**Version:** 2.2.8

**Conda Channel:** bioconda

#### Summary:
Fast and sensitive read alignment



### cufflinks
**Version:** 2.2.1

**Conda Channel:** bioconda

#### Summary:
Transcriptome assembly and differential expression analysis for RNA-Seq.



### fastqc
**Version:** 0.11.5

**Conda Channel:** bioconda

#### Summary:
A quality control tool for high throughput sequence data.



### gatk
**Version:** 3.6

**Conda Channel:** bioconda

#### Summary:
The full Genome Analysis Toolkit (GATK) framework, license restricted.



### gencore_rnaseq_docs
**Version:** 1.0

**Conda Channel:** nyuad-cgsb

#### Summary:




### hisat2
**Version:** 2.0.4

**Conda Channel:** bioconda

#### Summary:
graph-based alignment of next generation sequencing reads to a population of genomes



### htseq
**Version:** 0.6.1.post1

**Conda Channel:** bioconda

#### Summary:
A framework to process and analyze data from high-throughput sequencing (HTS) assays



### kallisto
**Version:** 0.43.0

**Conda Channel:** bioconda

#### Summary:
Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.



### perl-biox-workflow-plugin-filedetails
**Version:** 0.11

**Conda Channel:** bioconda

#### Summary:
Get metadata for files in directories processed by BioX::Workflow



### perl-biox-workflow-plugin-fileexists
**Version:** 0.13

**Conda Channel:** bioconda

#### Summary:
a plugin to BioX::Workflow



### perl-biox-workflow
**Version:** 1.10

**Conda Channel:** bioconda

#### Summary:
A very opinionated template based workflow writer.



### perl-hpc-runner-command
**Version:** 3.1.1

**Conda Channel:** bioconda

#### Summary:




### picard
**Version:** 2.5.0

**Conda Channel:** bioconda

#### Summary:




### prinseq
**Version:** 0.20.4

**Conda Channel:** bioconda

#### Summary:
PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data



### r-base
**Version:** 3.3.1

**Conda Channel:** r

#### Summary:




### r-cairo
**Version:** latest

**Conda Channel:** r

#### Summary:
This package provides a Cairo graphics device that can be use to create high-quality vector (PDF, PostScript and SVG) and bitmap output (PNG,JPEG,TIFF), and high-quality rendering in displays (X11 and Win32). Since it uses the same back-end for all output, copying across formats is WYSIWYG. Files are created without the dependence on X11 or other external programs. This device supports alpha channel (semi-transparent drawing) and resulting images can contain transparent and semi-transparent regions. It is ideal for use in server environments (file output) and as a replacement for other devices that don't have Cairo's capabilities such as alpha support or anti-aliasing. Backends are modular such that any subset of backends is supported.



### r-essentials
**Version:** latest

**Conda Channel:** r

#### Summary:
Some essential packages for working with R



### samtools
**Version:** 1.3.1

**Conda Channel:** bioconda

#### Summary:
Tools for dealing with SAM, BAM and CRAM files



### seqtk
**Version:** 1.2

**Conda Channel:** bioconda

#### Summary:
Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format



### star
**Version:** 2.5.2a

**Conda Channel:** bioconda

#### Summary:
An RNA-seq read aligner.



### stringtie
**Version:** 1.3.0

**Conda Channel:** bioconda

#### Summary:
Transcriptome assembly and quantification for RNA-seq



### tophat
**Version:** 2.1.0

**Conda Channel:** bioconda

#### Summary:
A spliced read mapper for RNA-Seq



### trimmomatic
**Version:** 0.36

**Conda Channel:** bioconda

#### Summary:
Genetic variant annotation and effect prediction toolbox



### vcflib
**Version:** 1.0.0_rc1

**Conda Channel:** bioconda

#### Summary:
Command-line tools for manipulating VCF files



### vcftools
**Version:** 0.1.14

**Conda Channel:** bioconda

#### Summary:
A set of tools written in Perl and C++ for working with VCF files.



